.gitignore

```
@@ -1,2 +1,23 @@
-__pycache__/
+# Binaries for programs and plugins
+*.exe
+*.exe~
+*.dll
+*.so
+*.dylib
+bin
+cmd/bin
+.vscode
+perf
+*.pdf
 
+# Test binary, built with `go test -c`
+*.test
+
+# Output of the go coverage tool, specifically when used with LiteIDE
+*.out
+
+# Dependency directories (remove the comment below to include it)
+# vendor/
+
+# .DS_Store
+.DS_Store
```

```suggestion
.DS_Store

```

nit: add newline to EOF

You'll also want to make sure that anything matching .gitignore that was already committed is removed e.g., https://github.com/cisco-open/jalapeno/blob/0a94ef83c5c95705d5a566c82a480accdd995c97/.DS_Store